crates/yttrium/src/uniffi_compat/sign/client.rs

```diff
@@ -194,7 +194,7 @@ impl SignClient {
             let mut client = self.client.lock().await;
             client.pair(&uri).await?
         };
-        Ok(proposal.into())
+        Ok(proposal.0.into())
```

Comment:
Are we passing VerifyContext here?

Reply:
`.0` is the proposal, `.1` is VerifyContext

We can refactor the API